drivers/hmis_external_apis/public/schemas/form_definition.json

```diff
@@ -203,83 +821,273 @@
+        ]
+      }
+    },
+    "genericInputFormNode": {
+      "$comment": "Generic form input",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/$defs/baseInputFormNode"
         },
-        "link_id": { "type": "string" },
-        "prefix": { "type": "string" },
-        "brief_text": { "type": "string" },
-        "text": { "type": "string" },
-        "prefill": { "type": "boolean" },
-        "readonly_text": { "type": "string" },
-        "helper_text": { "type": "string" },
-        "required": { "type": "boolean" },
-        "assessment_date": { "type": "boolean" },
-        "read_only": { "type": "boolean" },
-        "hidden": { "type": "boolean" },
-        "warn_if_empty": { "type": "boolean" },
-        "pick_list_reference": { "type": "string" },
-        "mapping": { "$ref": "#/definitions/mapping" },
-        "project_types_excluded": { "type": "array", "items": { "type": "string" } },
-        "enable_when": { "type": "array", "items": { "$ref": "#/definitions/enable_when" } },
-        "rule": { "$ref": "#/definitions/rule" },
-        "disabled_display": { "type": "string" },
-        "bounds": { "type": "array", "items": { "$ref": "#/definitions/bounds" } },
-        "repeats": { "type": "boolean" },
-        "size": { "type": "string" },
-        "service_detail_type": { "type": "string" },
-        "pick_list_options": {
-          "type": "array",
-          "items": { "$ref": "#/definitions/pick_list_option" }
+        {
+          "$ref": "#/$defs/textLabelRequired"
+        }
+      ]
+    },
+    "baseInputFormNode": {
+      "$comment": "Generic form input",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/$defs/baseFormNode"
         },
-        "initial": { "type": "array", "items": { "$ref": "#/definitions/initial" } },
-        "autofill_values": {
-          "type": "array",
-          "items": { "$ref": "#/definitions/autofill_values" }
+        {
+          "$ref": "#/$defs/dataCollectedAbout"
         },
-        "component": {
+        {
+          "properties": {
+            "hidden": {
+              "type": "boolean"
+            },
+            "prefix": {
+              "type": "string"
+            },
+            "brief_text": {
+              "type": "string"
+            },
+            "text": {
+              "type": "string"
+            },
+            "readonly_text": {
+              "type": "string"
+            },
+            "helper_text": {
+              "type": "string"
+            },
+            "required": {
+              "type": "boolean"
+            },
+            "assessment_date": {
+              "type": "boolean"
+            },
+            "read_only": {
+              "type": "boolean"
+            },
+            "warn_if_empty": {
+              "type": "boolean"
+            },
+            "mapping": {
+              "$ref": "#/$defs/mapping"
+            },
+            "project_types_excluded": {
```

Comment:
Do you know what project_types_excluded is? I don't remember where this came from, and I don't see it in existing code/forms.

Reply:
not sure, yeah it seems unused. I'll take it out